src/System.Private.ServiceModel/tests/Stress/SharedPoolsOfWCFObjects/project.json

```diff
@@ -1,37 +1,30 @@
-{
-  "version": "1.0.0-*",
+﻿{
+  "version": "1.0.0-23719",
   "description": "",
-  "authors": [
-    ""
-  ],
-  "tags": [
-    ""
-  ],
+  "authors": [ "" ],
+  "tags": [ "" ],
   "projectUrl": "",
   "licenseUrl": "",
+
   "dependencies": {
-    "System.Net.Http": "4.0.1-rc2-23712",
-    "System.Net.Sockets": "4.0.11-rc2-23712",
-    "System.ServiceModel.NetTcp": "4.0.1-rc2-23712",
-    "System.ServiceModel.Http": "4.0.11-rc2-23712",
-    "System.ServiceModel.Primitives": "4.1.0-rc2-23712",
-    "System.ServiceModel.Security": "4.0.1-rc2-23712",
-    "System.Console": "4.0.0-rc2-23712",
-    "System.Collections": "4.0.11-rc2-23712",
-    "System.Linq": "4.0.1-rc2-23712",
-    "System.Threading": "4.0.11-rc2-23712",
-    "System.Diagnostics.Debug": "4.0.11-rc2-23712",
-    "System.Collections.Specialized": "4.0.1-rc2-23712",
-    "System.Private.ServiceModel": "4.0.1-rc2-23712",
-    "System.Diagnostics.Tools": "4.0.1-rc2-23712",
-    "System.ServiceModel.Duplex": "4.0.1-rc2-23712"
```

Comment:
Recommend option 2 also.  It is vital each PR be atomic, so any product wide touch to our build.proj and project.json's must contain only those changes and nothing else.  For example, we need the option to update the release branch with only the project.json touches.  I also think the version will be 23721 or more by the time the next PR happens.